SU2_CFD/src/drivers/CDriver.cpp

```diff
@@ -3757,14 +3757,18 @@ bool CFluidDriver::Monitor(unsigned long ExtIter) {
     case DISC_ADJ_FEM_EULER: case DISC_ADJ_FEM_NS: case DISC_ADJ_FEM_RANS:
       StopCalc = integration_container[ZONE_0][INST_0][ADJFLOW_SOL]->GetConvergence(); break;
   }
+  
+  /*--- Set StopCalc to true if max. number of iterations has been reached ---*/
+  
+  StopCalc = (ExtIter == Max_Iter - 1);
 
   return StopCalc;
```

Comment:
The upper `StopCalc` is obsolete now as it is overwritten with the code addition anyway.